### PR TITLE
Clarify Redis package vs SignalR version

### LINKED ISSUE #9919: Why Microsoft.AspNetCore.SignalR.StackExchangeRedis?
In  this documentation you are referring to Microsoft.AspNetCore.SignalR.StackExchangeRedis where in fact there also exists another (official?) package Microsoft.AspNetCore.SignalR.Redis. Maybe at least mention the other package as an alternative?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c548355c-d88f-b941-0622-56732973123b
* Version Independent ID: bddd7575-422d-4342-af48-f6b0979fb69a
* Content: [Redis backplane for ASP.NET Core SignalR scale-out](https://docs.microsoft.com/en-us/aspnet/core/signalr/redis-backplane?view=aspnetcore-2.2#feedback)
* Content Source: [aspnetcore/signalr/redis-backplane.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/signalr/redis-backplane.md)
* Product: **aspnet-core**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Add text to clarify that StackExchangeRedis is for 2.2, non-SO Redis is for 2.1.
Fixes #9919

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Technically AspNetCore.SignalR.Redis is for 2.1 and 2.2, and we are removing it in 3.0

File: aspnetcore/signalr/redis-backplane.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
 
 ::: moniker range="= aspnetcore-2.1"
 
-* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.Redis` NuGet package.
+* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.Redis` NuGet package. (This package is for ASP.NET Core 2.1; there is also a `Microsoft.AspNetCore.SignalR.StackExchanggeRedis` package, which is for ASP.NET Core 2.2 and later.)
```
Comment (BrennanConroy) [inline aspnetcore/signalr/redis-backplane.md]: "StackExchanggeRedis" => "StackExchangeRedis"

File: aspnetcore/signalr/redis-backplane.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
 
 ::: moniker range="> aspnetcore-2.1"
 
-* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.StackExchangeRedis` NuGet package.
+* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.StackExchangeRedis` NuGet package. (This package is for ASP.NET Core 2.2 and later; there is also a `Microsoft.AspNetCore.SignalR.Redis` package, which is available for ASP.NET Core 2.1 and 2.2 but will not be supported in 3.0.)
```
Comment (BrennanConroy) [inline aspnetcore/signalr/redis-backplane.md]: Maybe something more like:
"This package depends on StackExchange.Redis 2.X.X. There is also the Microsoft.AspNetCore.SignalR.Redis package which depends on StackExchange.Redis 1.X.X, this package will not be shipping in 3.0. Choose the one appropriate for your applications dependencies."

File: aspnetcore/signalr/redis-backplane.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
 
 ::: moniker range="= aspnetcore-2.1"
 
-* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.Redis` NuGet package.
+* In the SignalR app, install the `Microsoft.AspNetCore.SignalR.Redis` NuGet package. (This package is for ASP.NET Core 2.1. It can also be used with 2.2, but it will not be supported in 3.0. The `Microsoft.AspNetCore.SignalR.StackExchangeRedis` package is for ASP.NET Core 2.2 and later.)
```
[the remark this answers is not in the document]
Comment (tdykstra) [reply aspnetcore/signalr/redis-backplane.md]: If we don't say anything, people will wonder if the other package is usable for 2.1 and we just forgot to mention it.

**Pull request merged**